package.json

```diff
@@ -61,5 +64,30 @@
   },
   "dependencies": {
     "github-contributors-list": "https://github.com/woocommerce/github-contributors-list/tarball/master"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "linters": {
+      "*.php": [
+        "php -d display_errors=1 -l",
+        "composer run-script phpcs"
```

Comment:
`composer run-script phpcs` calls PHPCS using the following command `phpcs -s -p` whereas `woocommerce-git-hooks` uses `phpcs --encoding=utf-8 -n -p`. `--enconding=utf8-8` is not necessary as it is the default value, but maybe we should keep `-n` to keep ignoring warnings and to perform the same check in the pre-commit use as the check we perform on Travis?